proposal/prefixed-parent-selector.md

```diff
@@ -0,0 +1,58 @@
+# Prefixed Parent Selector: Draft 2
+
+[Issue](https://github.com/sass/sass/issues/1425)
+
+## Background
+
+When transitioning from Less or Stylus, or just learning Sass, nesting is common.
+
+Everything works just as expected until a `&` is in the beginning of a nested selector.
+
+`"&" may only used at the beginning of a compound selector.` 
+
+Considering that there's been quite a few issues regarding this, and that they keep comming up, 
+and that people expect this to work, I think this would be a valuable addition to Sass.
```

Comment:
This doesn't do an effective job of making the case for why users want this behavior and why they expect this to work. I'd really like to see at least one concrete use-case written out here, as well as some discussion of users' expectation of symmetry between parent selectors with a suffix and those with a prefix.

Reply:
Attempted to address this in: https://github.com/sass/sass/pull/2723/commits/7631df1dfd0e4b5ca6b134834d9e1d9ded17c7d9